about-pses/example-pse.md

```diff
@@ -58,20 +130,24 @@ Try not to pose questions that are too specific to the implementation. For examp
+1. **What should happen if both users have invalid input?**
+    - While both players providing invalid input could be considered a tie, I don't want the function to act as though everything was fine if an input was invalid.
+    - I'll return `None` if either player has invalid input. This way, the caller can detect that something went wrong and decide how to handle it. A drawback of this approach is that the caller can't tell _which_ input was invalid, only that one of them was.
+2. **Does capitalization matter?**
+    - I want to know whether we need to tell the difference between “Rock” and “rock”
+    - I will assume that all input will be lowercase as shown in the Rock, Paper, Scissors rules on Learn
+3. **Does extra whitespace / punctuation matter?**
+    - I want to know whether inputs like " rock " or "rock!" are valid or should be rejected.
+    - I will assume we are getting clean input without any extra characters as shown in the Rock, Paper, Scissors rules on Learn
+4. **Should anything be printed to the console?**
+    - I know that we want to return the result of the match based on the problem statement, but I want to know if we should also print the result for the user to see.
+    - Since the output examples do not mention anything printed out, I will assume that we are only returning the result as a string
+5. **I'm familiar with a variant of rock, paper, scissors that includes additional throws (lizard and Spock). Should I keep my approach focused on the basic rules, or should I use an implementation that could be extended to game variants?**
+    - I want to know if my implementation needs to be flexible enough to extend it with additional game rules.
+    - While I could write a helper function that accepts a data structure that encodes the valid moves and what beats what, I'll assume that I should focus on the basic rules for now. This way, I can get a working solution faster, and if I need to extend the game later, I can refactor my code to be more flexible.
+6. **When friends use rock, paper, scissors to make decisions, there are often more than two players. Should I keep my approach focused on the basic rules for 2 players?**
+    - I want to know if I should I use an implementation that could be extended to support multiple players simultaneously.
+    - While I could write a helper function that accepts a list of moves made by all players, the multiplayer version introduces a few complications into resolving a round that I'd like to avoid for now. I'll assume that I should focus on the basic rules for now. This way, I can get a working solution faster, and if I need to extend the game later, I can refactor my code to be more flexible.
```

Comment:
I like that, it feels a little less vague and focuses in on where we want students putting effort. How does this sound to incorporate that language?

```suggestion
    - While I could write a helper function that accepts a list of moves made by all players, the multiplayer version increases the scope of the problem. I'll assume that I should focus on the basic rules for now. This way, I can get a working solution faster, and if I need to extend the game later, I can refactor my code to be more flexible.
```

Reply:
Yeah! That looks good to me.